main.tf

```diff
@@ -1,18 +1,13 @@
-resource "null_resource" "ansible" {
-  triggers = {
-    command = "ansible-playbook ${var.dry_run ? "--check --diff" : ""} ${join(" ", var.arguments)} -e ${join(" -e ", var.envs)} ${var.playbook}"
-  }
-}
-
 resource "null_resource" "provisioner" {
   count = "${signum(length(var.playbook)) == 1 ? 1 : 0}"
 
   triggers {
     default = "${sha256(file(var.playbook))}"
+    command = "ansible-playbook ${var.dry_run ? "--check --diff" : ""} ${join(" ", var.arguments)} -e ${join(" -e ", var.envs)} ${var.playbook}"
```

Comment:
Let's call this something other that `default`.... maybe `signature`, or `serial` or `nonce`